custom_connectors/none/two_checkout.rb

```diff
@@ -1034,11 +1030,7 @@
       end,
 
       webhook_notification: lambda do |_input, payload, _e_i_s, _e_o_s, _headers|
-        if payload['HASH'].present? && payload['DATE_UPDATED'].present?
-          payload
-        else
-          {}
-        end
+        payload || {}
       end,
 
       dedup: lambda do |_event|
```

Comment:
The Dedup block should be made up of the `HASH` and `DATE_UPDATED` values to be considered unique.